handler/src/test/java/io/netty/handler/ssl/SSLEngineTest.java

```diff
@@ -1009,7 +1009,7 @@ private static void verifyApplicationLevelProtocol(Channel channel, String expec
         SslHandler handler = channel.pipeline().get(SslHandler.class);
         assertNotNull(handler);
         String appProto = handler.applicationProtocol();
-        assertEquals(appProto, expectedApplicationProtocol);
+        assertEquals(expectedApplicationProtocol, appProto);
```

Comment:
I did this change as it was using the wrong order here so the error message was confusing